docs/content/docs/operations/helm.md

```diff
@@ -107,6 +107,28 @@ The webhook can be disabled during helm install by passing the `--set webhook.cr
 The operator supports watching a specific list of namespaces for FlinkDeployment resources. You can enable it by setting the `--set watchNamespaces={flink-test}` parameter.
 When this is enabled role-based access control is only created specifically for these namespaces for the operator and the jobmanagers, otherwise it defaults to cluster scope.
 
+Note, when working with webhook in a specified namespace, users should pay attention to the definition of `namespaceSelector.matchExpressions` in `webhook.yaml`. Currently, the default implementation of webhook relies on the `kubernetes.io/metadata.name` label to filter the validation requests
```

Comment:
nit: we could use `<span class="label label-info">Note</span>`.

Reply:
It does look nicer:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/80749729/170175942-a412af6f-e2b6-4ddf-adf8-2283984fedd0.png">